connectors/utils.py

```diff
@@ -465,9 +465,9 @@ def try_put(self, coroutine, result_callback=None):
             return self._add_task(coroutine, result_callback=result_callback)
         return None
 
-    async def join(self):
+    async def join(self, raise_on_error=False):
         """Wait for all tasks to finish."""
-        await asyncio.gather(*self.tasks, return_exceptions=True)
+        await asyncio.gather(*self.tasks, return_exceptions=(not raise_on_error))
```

Comment:
`return_exceptions=True` feels like a bad default, but at least something that should be controllable. `gather()`'s own default is `return_exceptions=False`, so IDK why we'd want to have the opposite.

I thought that making the parameter more explicit (`raise_on_errror`) might help clarify what this does.

Reply:
How does this practically impact us now? Will it change any behaviour?

Reply:
nope. I should add that, good point.